starter-code/js/index.js

```diff
@@ -11,7 +11,33 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
+  var totalCost = 0;
+  for (var x = 0; x < document.querySelectorAll('.product').length ; x ++) {
```

Comment:
Better create an array variable with document.querySelectorAll('.product') and iterate through the array (it is faster).